src/components/Votes.jsx

```diff
@@ -0,0 +1,45 @@
+import { useState, useEffect } from "react";
+
+import { upVoteCount } from "../utils/api";
+import { downVoteCount } from "../utils/api";
+
+const Votes = ({ article_id, setArticle, article, articles }) => {
+  const [voted, setVoted] = useState(false);
+  const [tempVotes, setTempVotes] = useState(null);
+  const [num, setNum] = useState(0);
+  const [err, setErr] = useState("there is a problem")
```

Comment:
You've got this err state so use it! It would be great if when a user clicks the vote button when offline, they get some sort of feedback, rather than having to refresh the page to see that the vote hasn't persisted.